src/screens/ExplainSheet.js

```diff
@@ -98,8 +98,8 @@ export const explainers = {
   gas: {
     emoji: '⛽️',
     extraHeight: 2,
-    text: GAS_EXPLAINER,
-    title: 'Ethereum network fee',
+    text: gasExplainer(network),
```

Comment:
hm this feels strange, having to pass a network for only one of the params in the object. I guess the only way to make it cleaner would be to have a separate method to act according to a `network` which would only be used when `network` is present otherwise use this object or something like that ... if we have some guidelines for this cc @jinchung

Reply:
def get what u mean - there are a couple options but none that are quick and clever